ui/lobby/src/main.ts

```diff
@@ -41,15 +41,21 @@ const layoutHacks = () =>
         if (cols > 2) $('.lobby .lobby__timeline').appendTo('.lobby__side');
         else $('.lobby__side .lobby__timeline').appendTo('.lobby');
       }
+
+      // fake tv for debugging layout, ui/build with -d flag
       if (site.debug && !this.querySelector('.lobby__tv')) {
+        const ds = document.body.dataset;
         const tv = $as<HTMLElement>(
-          `<div class="lobby__tv"><span class="text">Fake TV</span><span class="mini-board"
-          data-state="3R1r1k/pp4p1/2n1Q1bp/1Bp5/PqN4P/2b2NP1/1P4P1/2K4R,black,d1d8"/></div>`,
+          `<div class="lobby__tv">
+            <a href="/tv" class="mini-game mini-game-abcd1234 standard is2d">
+              <span class="mini-game__player"><span class="mini-game__user"><span class="utitle" title="Candidate Master">CM</span>&nbsp;Tester1<img class="uflair" src="${ds.assetUrl}/assets/______2/flair/img/activity.lichess-horsey.webp"><span class="rating">2649</span></span><span class="mini-game__clock mini-game__clock--black clock--run" data-time="60">0:26</span></span>
+              <span id="fake-tv" data-state="3R1r1k/pp4p1/2n1Q1bp/1Bp5/PqN4P/2b2NP1/1P4P1/2K4R,black,d1d8"></span>
+              <span class="mini-game__player"><span class="mini-game__user"><span class="utitle" title="FIDE Master">FM</span>&nbsp;tester2<img class="uflair" src="${ds.assetUrl}/assets/______2/flair/img/activity.lichess-berserk.webp"><span class="rating">2760</span></span><span class="mini-game__clock mini-game__clock--white" data-time="60">0:19</span></span>
+            </a>
+          </div>`,
```

Comment:
is there a chance that the TS build process could erase this from the prod build?

Reply:
I was hoping, with the code splitting thing and overall ui build improvements, that the static code analysis could act upon `if (site.debug` in some way.
Only goes to show how out of the loop I am with ui builds.

Reply:
How about adding one extra `<script src=` on all pages in dev mode? Then we can stuff in there all that kind of stuff.

Reply:
Sure, that's good too. A conf setting I like.

Reply:
Re-use `net.isProd` or should it be its own explicit config?